webapp/channels/src/components/drafts/drafts_link/drafts_link.tsx

```diff
@@ -51,8 +51,11 @@ function DraftsLink() {
     const draftCount = useSelector(getDraftsCount);
     const teamId = useSelector(getCurrentTeamId);
     const teamScheduledPostCount = useSelector((state: GlobalState) => getScheduledPostsByTeamCount(state, teamId, true));
+    const isScheduledPostEnabled = useSelector(isScheduledPostsEnabled);
 
-    const itemsExist = draftCount > 0 || teamScheduledPostCount > 0;
+    const hasDrafts = draftCount > 0;
+    const hasScheduledPosts = teamScheduledPostCount > 0;
+    const itemsExist = isScheduledPostEnabled ? hasDrafts || hasScheduledPosts : hasDrafts;
```

Comment:
this can be simplified to: `const itemsExist = hasDrafts || (isScheduledPostEnabled && hasScheduledPosts);` .

Reply:
DOne.